components/campaign/donation/wallet/details.tsx

```
@@ -9,13 +9,14 @@ import { CampaignDonationWalletTip } from './tip';
 import { CampaignDonationWalletProcess } from './process';
 import { CampaignDonationAnonymous } from '../anonymous';
 import { VisibilityToggle } from '@/components/visibility-toggle';
+import { FeeInformation } from '@/components/shared/fee-information';
 
 export function CampaignDonationWalletDetails({
   campaign,
 }: {
   campaign: DbCampaign;
 }) {
-  const { isProcessingPayment, clearDonation } = useDonationContext();
+  const { isProcessingPayment, clearDonation, amount } = useDonationContext();
```

ESLint

[error] 'amount' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)